### PR TITLE
navbar for std.experimental.logger

### PR DESCRIPTION
**Pull request by CyberShadow**
Alternative with separate "std.experimental" menu

### CODE REVIEW

File: std_navbar-prerelease.ddoc [hunk 1]
```diff
@@ -121,6 +121,16 @@ $(DIVID cssmenu, $(UL
 		  $(MODULE3 core, sync, semaphore)
 		)
 	  )
+	$(MENU_W_SUBMENU $(TT std.experimental))
+	  $(ITEMIZE
+		$(MODULE3 std, experimental, logger, package),
```
Comment (burner) [inline std_navbar-prerelease.ddoc]: shouldn't this be MODULE4

Comment (CyberShadow) [inline std_navbar-prerelease.ddoc]: No, the group caption should say "logger", not "package". Look e.g. at how std.container is presented in the navigation.

Comment (burner) [inline std_navbar-prerelease.ddoc]: I see

**Comment by burner** [comment]
thanks

**Comment by CyberShadow** [comment]
Um, the  idea was to merge either this on #865...

**Comment by CyberShadow** [comment]
I think putting it in the main "std" menu is a little better than as a separate menu.

The rationale being that a user who knows about std.experimental would start looking for the module's documentation under the "std" menu - they might not notice the separate menu way below, and wonder why `experimental` is not in its alphabetically-sorted place.

**Pull request merged**